Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+String path = 'src/'
+
+node('docker') {
+    ansiColor('xterm') {
+
+        stage('Checkout') {
+            checkout scm
+        }
+
+        stage('Pulling') {
+            sh 'docker pull jakzal/phpqa'
+        }
+
+        stage('Prepare directory') {
+            sh 'rm -rf build/logs'
```

Comment:
@dgrochowski we also improved this part:
`sh 'docker run --rm -v $(pwd):/project -w /project jakzal/phpqa chmod -R 777 build'` to be sur that dir is writable